sirius-valley/lerni-mobile
# [LER-BUG-35] Loading trivia screen

On this PR:
- Loading screen now loads participant names. 

Extra: 
- Some files on this pr only have prettier changes, quotes and spaces.

To test this fix: 
- There are several ways to test this fix
  1. Finish a questionnaire and play trivia, you should see your name loaded. 
  2. Finish a questionnaire, don't start the trivia go to trivia screen and start trivia from trivia card, you should see the loading screen with your name loaded.
  3. For this final test you need to users in the same program, i'm testing with monosteve123 and monosteve1237, in the program with id: 'programId 7'. This are the steps to access the challenge card:
   - Finish a questionnaire with one student
   -  don't start the trivia
   -  finish a questionnaire with another student in the same program and start the trivia, answer at least one question.
   - switch to the previous user and go to trivia screen, you should see the challenged card, start trivia and watch loading screen with the correct names loaded
   

https://github.com/sirius-valley/lerni-mobile/assets/9684898/2c684b8f-a3a5-4536-aac8-703d240d7497


https://github.com/sirius-valley/lerni-mobile/assets/9684898/dd89fbba-05bd-48b9-ae89-05f6068a4b23


https://github.com/sirius-valley/lerni-mobile/assets/9684898/7d02cb39-9522-42d9-ad5a-a6f6db75dab9
